pkg/generator/ingress_translator.go

```diff
@@ -109,15 +109,15 @@ func (translator *IngressTranslator) translateIngress(ctx context.Context, ingre
 				endpoints, err := translator.endpointsLister.Endpoints(split.ServiceNamespace).Get(split.ServiceName)
 				if apierrors.IsNotFound(err) {
 					translator.logger.Warnf("Endpoints '%s/%s' not yet created", split.ServiceNamespace, split.ServiceName)
-					break
+					return nil, nil
```

Comment:
Looks good :+1:
Should we `return nil, err` and delete the warning above ? :thinking:

Reply:
Yes, sounds good. Updated.